website/documentation/050-tutorials/content/howto/shell-to-node/_index.md

```diff
@@ -0,0 +1,333 @@
+Usually the problematic instance can be identified as the respective `Machine` CRD is `pending`.
+The `Machine` already contains the instance / node name in the .status, obtain it via:
+`k get machine <machine-name> -o json | jq -r .status.node`
+
+This is all the information needed to go ahead and use `gardenctl ssh` to get a shell to the node (you can skip to the section [Gardenctl ssh](#gardenctl-ssh) below). 
+
+Additionally we can identify the Cloud Provider, the Cloud Provider specific identfier of the instance and the region via: 
+
+`
+k get machine <machine-name> -o json | jq -r .spec.providerID // e.g aws:///eu-north-1/i-069733c435bdb4640
+`
+In this case: Cloud Provider `aws` with the ec2 instance-id `i-069733c435bdb4640` in region `eu-north-1`.
+Additonal information about the instance can be found in the associated `<AWS/GCP/..>MachineClass` and can be obtained via the Cloud Provider specific dashboard & CLI.
+
+
+## Gardenctl ssh
+
+Using the node name of the problematic instance, we can use the `gardenctl ssh` command to get a shell to the Cloud Provider instance via an automatically set up bastion.
+The steps that `gardenctl` performs for each Cloud Provider [can be found here]( https://github.com/gardener/gardenctl/blob/10a537942b94234914758c0f6d053dc1cf218ecd/pkg/cmd/ssh_aws.go#L53-L52).
+
+
+
+Use the following commands:
+
+```
+gardenctl target garden <target-garden>
+
+gardenctl target shoot <target-shoot>
+
+gardenctl ssh <target-node>
+```
+
+After waiting for the successful set-up of the bastion, this should open a shell on the target instance.
+
+
+## Ssh via manually created Bastion (AWS)
+
+In case you are not using `gardenctl` or want to control the bastion instance yourself, you can manually set-up the bastion. 
+The steps described here are generally the same as when using `gardenctl`.
+The setup of a bastion host are different from Provider to Provider but generally involves the following steps:
+- Open port 22 on the problematic instance.
+- Create an instance / VM in a public subnet (bastion instance needs to have public ip adress).  
+- Set-up security groups, roles and open port 22 for the bastion instance.
+
+The following demonstrated the setup of a bastion on AWS.
+
+**Prerequisites:**
+- Obtain target instance-id (see [here](#identifying-the-problematic-instance))
+- Obtain the VPC ID of the Shoot
+- Make sure that port 22 on the target instance is open (default for Gardener deployed instances)
+    - Extract security group via `aws ec2 describe-instances --instance-ids <instance-id>`
+    - Check for rule that allows inbound connections on port 22 via `aws ec2 describe-security-groups --group-ids=<security-group-id>`
+    - If not available create via: `aws ec2 authorize-security-group-ingress --group-id <security-group-id>  --protocol tcp --port 22 --cidr 0.0.0.0/0` 
+
+### Create the Bastion security group
```

Comment:
title case: 
```suggestion
### Create the Bastion Security Group
```

Reply:
Consider using list formatting for this section. It's a step by step guide and using newlines you format it close to list anyway. The trick with lists when you provide more than one sentence per bullet, but a paragraph is to use tripple spaces and no blank lines. That binds together sentences in a section even if they do not belong to the same paragraph. See if that works for you or not.